doc/user/content/ingest-data/postgres/self-hosted.md

```diff
@@ -274,37 +307,94 @@ start by selecting the relevant option.
 
 {{< tab "Allow Materialize IPs">}}
 
-1. In the SQL client connected to Materialize, use the [`CREATE
-   SECRET`](/sql/create-secret/) command to securely store the password for the
-   `materialize` PostgreSQL user you created
-   [earlier](#2-create-a-publication-and-a-replication-user):
 
-    ```mzsql
-    CREATE SECRET pgpass AS '<PASSWORD>';
-    ```
+1. Click **+ Create New** and select **Source** to open the **Create a Source**
+   panel. From the **Create a Source** panel, select **PostgreSQL**.
 
-1. Use the [`CREATE CONNECTION`](/sql/create-connection/) command to create a
-   connection object with access and authentication details for Materialize to
-   use:
+   ![Image of the Create New Source start](/images/console/console-create-new/postgresql/create-new-source-start.png "Create New Source start")
 
-    ```mzsql
-    CREATE CONNECTION pg_connection TO POSTGRES (
-      HOST '<host>',
-      PORT 5432,
-      USER 'materialize',
-      PASSWORD SECRET pgpass,
-      SSL MODE 'require',
-      DATABASE '<database>'
+
+1. From the **Configure a connection** panel, you can choose to:
+
+   - Use an **Existing** connection.
+
+   - Create a **New** connection.
+
+   For a new connection, specify the following connection information and click **Continue**:
+
+   | Field | Description |
+   | ----- | ----------- |
+   | **Name** | A name for the connection. |
+   | **Schema** | A Materialize database and schema for the connection. |
```

Comment:
Could we separate these out into the different sections, like in the console? It's pretty confusing otherwise, since these two options refer to Materialize, and the options below to PostgreSQL.

Reply:
Also, not sure how useful it is to document each field in the documentation. If it's unclear what the field refers to, it should be annotated in the UI itself, which can then point to the [`CREATE CONNECTION` reference documentation](https://materialize.com/docs/sql/create-connection/#postgres-options). This also makes it more cumbersome to make changes to the source creation workflow in the console, and unnecessarily creates multiple definitions for the same concepts we already document elsewhere for the underlying commands.

Reply:
Personally agree with you on limited usefulness on each field (and definitely, difficult to maintain + again with you 💯 on it should be in the UI).  While I have found in the past that some readers do seem to prefer having these,  I'll see if I can make it less specific fields in the patch (since, later on, as we get more users -- if it turns out we need it here, we can add it back here).